ansible/README.md

```diff
@@ -1,11 +1,32 @@
 # Ansible
 
 This directory contains all [Ansible](http://docs.ansible.com/) playbooks and
-files to setup, configure and deploy services automatically.
+files required to setup services automatically.
+
+## Installing Ansible
+
+If you still do not have Ansible installed in your system, setup EPEL yum
+repository and run:
+
+`yum install -y ansible`
 
 ## Running Ansible playbooks
 
-You can run Ansible playbooks by executing `ansible-playbook` command:
+First, update the host in the hosts.ini section which corresponds to the
+playbook which will be executed (eg if you will execute the jenkins-master
+playbook, update the host in [jenkins-master] section in hosts.ini)
+
+Then, update the variables values in vars.yaml. If you are creating a production
+Jenkins server, provide the production GitHub organization (open-power-host-os) in
```

Comment:
Other organization may replicate the setup, and they will not want to use our organization's name (open-power-host-os).